codice/ddf
# DDF-6166 Refactor Security Features File

Rename platform-session-invalidator to platform-session-invalidator-api
Rename security-logout-service to security-servlet-logout-api
Rename security-servlet-web-socket to security-servlet-web-socket-api
Rename session-management-service to session-management-api
Refactor security features

#### What does this PR do?
Simplifies the security feature file. This change will make the feature file easier to understand and maintain.

#### Who is reviewing it? 
@blen-desta 
@garrettfreibott 

#### How should this be tested?
[See the tests steps on the Alliance branch](https://github.com/codice/alliance/pull/829).

#### What are the relevant tickets?
Fixes: #6166

#### Checklist:

#### Notes on Review Process
Please see [Notes on Review Process](https://codice.atlassian.net/wiki/spaces/DDF/pages/71946981/Pull+Request+Guidelines) for further guidance on requirements for merging and abbreviated reviews. 

#### Review Comment Legend:
- ✏️ (Pencil) This comment is a nitpick or style suggestion, no action required for approval. This comment should provide a suggestion either as an in line code snippet or a gist. 
- ❓ (Question Mark) This comment is to gain a clearer understanding of design or code choices, clarification is required but action may not be necessary for approval.
- ❗ (Exclamation Mark) This comment is critical and requires clarification or action before approval.
